db/migrate/20240520060901_add_requirement_to_user_fields.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddRequirementToUserFields < ActiveRecord::Migration[7.0]
+  def up
+    add_column :user_fields, :requirement, :integer, null: false, default: 0
```

Comment:
We need to be careful when adding new columns with default values, because it will lock the entire table for the duration of the migration. This is a serious problem for huge tables.

After some research on our most heavyweight customers, we haven't found any that has more than a few custom user fields, so this should be safe.